src/main/java/org/terracotta/statistics/registry/StatisticRegistry.java

```diff
@@ -20,37 +20,35 @@
 import org.terracotta.context.query.Matcher;
 import org.terracotta.context.query.Matchers;
 import org.terracotta.statistics.OperationStatistic;
-import org.terracotta.statistics.Sample;
-import org.terracotta.statistics.SampledStatistic;
 import org.terracotta.statistics.StatisticType;
 import org.terracotta.statistics.Table;
 import org.terracotta.statistics.Time;
 import org.terracotta.statistics.ValueStatistic;
 
 import java.io.Serializable;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.EnumSet;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
+import java.util.function.LongSupplier;
 import java.util.function.Supplier;
 
-import static java.util.stream.Collectors.toList;
+import static java.util.Collections.singleton;
```

Comment:
I'm not a big fan of going crazy on static imports, makes it harder to know where the method is coming from

Reply:
True, but we'll have to live with that for all `import static org.terracotta.context.query.**` ;-)